py/ai/h2o/sparkling/ml/params/H2OCommonParams.py

```diff
@@ -17,6 +17,7 @@
 
 from h2o.utils.typechecks import assert_is_type
 from pyspark.ml.param import *
+from pyspark.ml.param import TypeConverters
```

Comment:
Isn't this line extra if we have the line above?

Reply:
Good catch, fixed